src/commands/dev/headers.rs

```diff
@@ -20,15 +45,12 @@ pub fn strip_response_headers_prefix(parts: &mut ResponseParts) -> Result<(), fa
     Ok(())
 }
 
-pub fn prepend_request_headers_prefix(parts: &mut RequestParts) {
-    let mut headers: HeaderMap = HeaderMap::new();
-
-    for header in &parts.headers {
-        let (name, value) = header;
-        let forward_header = format!("{}{}", HEADER_PREFIX, name);
-        let header_name = HeaderName::from_bytes(forward_header.as_bytes())
-            .unwrap_or_else(|_| panic!("Could not create header name for {}", name));
-        headers.insert(header_name, value.clone());
-    }
-    parts.headers = headers;
+fn set_response_status(parts: &mut ResponseParts) -> Result<(), failure::Error> {
+    let status = parts
+        .headers
+        .get("cf-ew-status")
+        .expect("Could not determine status code of response");
+    let status_vec: Vec<&str> = status.to_str()?.split(" ").collect();
```

Comment:
woah! why does the API return multiple status codes..?

Reply:
It doesn't! It returns it as a string like `404 Not Found` and we need to extract the `404`. Perhaps I should add a comment